packages/snet_cli/snet/snet_cli/metadata/service.py

```diff
@@ -167,6 +169,26 @@ def remove_group(self, group_name):
             if group["group_name"] == group_name:
                 self.m["groups"].remove(group)
 
+    def get_tags(self):
+        tags = []
+        if "tags" in self.m:
+            tags = self.m["tags"]
+        return tags
+
+    def add_tag(self, tag_name):
+        if (tag_name in self.m["tags"]):
+            print("the tag \"%s\" is already present" %
```

Comment:
String formatting structure is different. It should be the same across all code base.
print(f"the tag {tag_name} is already present").

Reply:
Done